objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/neuralNet/YoloProcessor.java

```diff
@@ -125,6 +126,40 @@ private void setup(Map<String, ?> neuralNet, String sourceName, String modelDire
        } else {
            log.debug("The threshold was not specified in the config. Using default threshold value of 0.5.");
        }
+       
+       // Get anchor values if they exist
+       if (neuralNet.get("anchors") instanceof List) {
+           List tempAnchorList = (List) neuralNet.get("anchors");
+           // Checking that there are 5 anchor pairs (10 elements total)
+           if (tempAnchorList.size() != 10) {
```

Comment:
magic numbers :-(

It would be better here to validate that the number of pairs matches YOLOClassifier.NUMBER_OF_BOUNDING_BOX(ES) * 2 (with an associated comment saying this.

In the future, we could, when we're reading the .meta file, expand this number to be variable, since (I think) it's provided in some way in the .meta file.